rasa_core/channels/slack.py

```diff
@@ -54,8 +54,15 @@ def send_attachment(self, recipient_id, attachment, message=""):
     def _convert_to_slack_buttons(buttons):
         return [{"text": b['title'],
                  "name": b['payload'],
+                 "value": b['payload'],
                  "type": "button"} for b in buttons]
 
+    @staticmethod
+    def _get_text_from_slack_buttons(buttons):
+        val = [bb['title'] for bb in buttons]
+        val = "".join(val)
+        return val
```

Comment:
```suggestion
```

Reply:
all changes are fine by me if you think it improves the readabilty and is still functionally identical

Reply:
yes should be the same, would be great if you can commit them, don't have push access to your fork